static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.spec.ts

```diff
@@ -452,6 +455,11 @@ describe('bids', () => {
 		expect(getBidders()).toEqual(['ix', 'trustx', 'adyoulike']);
 	});
 
+	test('should include Criteo if in target geolocation', () => {
+		shouldIncludeCriteo.mockReturnValue(true);
+		expect(getBidders()).toEqual(['ix', 'criteo', 'adyoulike']);
```

Comment:
It might be worth a follow-up checking why we don't have an explicit check for `adyoulike` but rather enable it and include it in each test